contents/tutorials/nextjs-analytics.md

```diff
@@ -1,51 +1,54 @@
 ---
 title: 'How to set up Next.js analytics, feature flags, and more'
-date: 2025-05-22
+date: 2025-08-13
 author:
   - ian-vanagas
 showTitle: true
 sidebar: Docs
-featuredTutorial: true
-featuredVideo: 'https://www.youtube-nocookie.com/embed/nSBjr1Sz18o'
 tags:
   - configuration
   - feature flags
   - persons
   - events
 ---
 
-Next.js is a popular web framework built on React. It provides optimizations and abstractions to help developers build fast and performant apps and sites.
+Next.js is one of the web's most popular frameworks. Built on React, it provides optimizations and abstractions to help developers build fast and performant apps and sites. 
 
-In this tutorial, we will:
+To make sure you get the most out of Next.js, you can use PostHog to track events, identify users, use feature flags, and more. To help you get started, this tutorial walks you through:
 
-1. Build a basic Next.js blog with user authentication
-2. Add PostHog to it
-3. Set up the features of PostHog like custom event capture, user identification, and feature flags
+1. Building a basic Next.js (pages router) blog with user authentication
```

Comment:
Should we specify in the title?

I think today, most new projects are app router (the direction vercel is pushing for. I know we have a separate post

Reply:
Yeah, I think I'm going to merge this then do another PR to swap them and redirect the app-router URL.